README.md

```diff
@@ -1 +1,50 @@
-# asyncapi-metrics-dashboard 
+# AsyncAPI Metrics Dashboard
+
+This Next.js 14 application parses AsyncAPI metrics from New Relic and displays them for relevant groups, providing a dashboard for viewing these statistics. It employs a Static Site Generation (SSG) model, pre-rendering pages at build time, and updates the data only once daily to ensure optimal performance.
```

Comment:
You mean working groups? in this case we can link to this discussion https://github.com/orgs/asyncapi/discussions/1037 to give more context.

Reply:
`viewing these statistics` -> `viewing metrics`

Reply:
Done.